dhcpv6/option_ntp_server.go

```diff
@@ -0,0 +1,58 @@
+package dhcpv6
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/u-root/uio/uio"
+)
+
+// NTPSuboptionSrvAddr is the value of NTP_SUBOPTION_SRV_ADDR according to RFC 5908.
+const NTPSuboptionSrvAddr byte = 1
```

Comment:
For consistency, I think this should be a type like in https://github.com/insomniacslk/dhcp/blob/d4b282fe8d4cf3f8023e01e0f17cb74c510ad228/dhcpv4/option_relay_agent_information.go#L48

Reply:
done